src/applications/disability-benefits/all-claims/Form526EZApp.jsx

```diff
@@ -16,4 +19,17 @@ export default function Form526Entry({ location, children }) {
       </ITFWrapper>
     </EVSSClaimsGate>
   );
+
+  if (!environment.isProduction()) {
```

Comment:
@cvalarida this is just to prevent the beta gate going into production—though I suppose there is no harm wrapping the feature in case anybody tries to go to the URL randomly; I'll get rid of the conditionals